Src/coffeescript/cql-execution/src/example/browser/cql4browsers.js

```diff
@@ -3626,7 +3626,7 @@
       ref = this.element;
       for (i = 0, len = ref.length; i < len; i++) {
         el = ref[i];
-        obj[el.name] = el.execute(ctx);
+        obj[el.name] = el.exec(ctx);
```

Comment:
When you do the browserify it seems to strip out the comments from the coffeescript.  Maybe there is switch we can use you preserve the comments?